docs/src/getting-started.rst

```diff
@@ -0,0 +1,83 @@
+Getting started
+===============
+
+A small introduction to all methods implemented in scikit-matter.
+For a detailed explaination, please look at the :ref:`selection-api`
+
+Features and Samples Selection
+------------------------------
+
+  .. include:: selection.rst
+     :start-after: marker-selection-introduction-begin
+     :end-before: marker-selection-introduction-end
+
+
+These selectors are available:
+
+* :ref:`CUR-api`: a decomposition: an iterative feature selection method based upon the
+  singular value decoposition.
+* :ref:`PCov-CUR-api` decomposition extends upon CUR by using augmented right or left
+  singular vectors inspired by Principal Covariates Regression.
+* :ref:`FPS-api`: a common selection technique intended to exploit the diversity of
+  the input space. The selection of the first point is made at random or by a
+  separate metric
+* :ref:`PCov-FPS-api` extends upon FPS much like PCov-CUR does to CUR.
+* :ref:`Voronoi-FPS-api`: conduct FPS selection, taking advantage of Voronoi
+  tessellations to accelerate selection.
+* :ref:`DCH-api`: selects samples by constructing a directional convex hull and
+  determining which samples lie on the bounding surface.
```

Comment:
This is very nice! I am only asking if we should also put this into the `selection.rst` as well.

Then this page only combines the API description with the examples.